src/selectors/teams.js

```diff
@@ -0,0 +1,27 @@
+import { createSelector } from 'reselect';
+
+/**
+ * @description Return team list from state
+ * @param {Object} state app state
+ */
+const getTeams = state => state.teams.list;
+
+/**
+ * @description selector for team array
+ * @return {Array} array of teams objects
+ */
+export const getTeamsArray = createSelector(getTeams, (teams) => {
+  const arr = [];
+  teams.forEach((team, id) => {
```

Comment:
This is good progress. 

Couple more changes.

1/ Rather than have the Teams accessible from Bottom Bar, have "Edit Teams..." option in the Team dropdown on the Edit Direct form. Clicking that should bring up list of Teams in a full-screen view without bottom bar. Back button left side in the header.

2/ There is no way to delete a team. The team form should have a delete button to delete the team. The team should not be able to be deleted if there are directs assigned to the team.

3/ In the list view, when sorting by Teams add list divider with the name of the team.